### ISSUE TITLE
Sensor Conflict 1.5.1 (X20S)

### ISSUE THREAD

**Issue by dWoolridge** [FrSkyRC/ETHOS-Feedback-Community#3614]
Ethos X20S
1.5.1, FCC 

Receivers are RX8R (ACCST F/W v2.1.0)

I have 2 RX8Rs bound to the transmitter, the first as Ch1-8 Tel ON, the second as Ch 9-16 Tel OFF.  I have 2 (older) FLVSS battery voltage monitors connected to the S.Port on the first receiver.

I get the "Sensor Conflict" announcement about every 30 seconds.

Sensors shown after auto discovery:
RSSI         (18 F101 ISRM Rx0)
RX            (18 F101 ISRM Rx0)
RxBatt      (18 F104  ISRM Rx0)
LIPO         (01 0300  ISRM Rx0)  [Renamed LIPO to LiFe1]
LIPO         (02 0300  ISRM Rx0)  [Renamed LIPO to LiFe2]

I had changed the ID of the second FLVSS using the FrSky ID changer.  

I ALSO DELETED THE "RX" Sensor (with duplicate ID:  18 F101).  The Sensor Conflict message remained even after power cycling Tx and Receivers with this sensor deleted.

**Comment by bsongis-frsky** [comment]
From your description, it seems your 2 LIPO sensors have the same appID (0300), no?

**Comment by dWoolridge** [comment]
@bsongis-frsky,

Hoping to learn something here...

I had this setup running in multiple models with a Taranis running OpenTx.  I have the FrSky Channel changer, but that only allows you to change the ID (first two numbers) -- the 01 and 02 for the LIPOs in my description.

How would I change the 0300 part of the ID on the FLVSS's?

**Comment by dWoolridge** [comment]
As I investigate further, I'm more confused on this.  I found a table documenting the "App ID" for FrSky sensors.  0x0300 is how the FLVSS sensor is identified in the FrSky protocol.  Wouldn't changing the sensor appID (0300) change the type of telemetry device that the transmitter would recognize?  Isn't this why we have the easily changeable ID for the sensor?

**Comment by dWoolridge** [comment]
Hello,

I don't mean to nag, but I was just wondering what the plan is for this issue.  I have continued moving models to my new X20S and run into this every time I have multiple FLVSS voltage sensors.

The data received for each FLVSS seems to be good, but the "Sensor Conflict" warning keeps repeating.

Please let me know if you need any additional information to help troubleshoot.

**Comment by bsongis-frsky** [comment]
@dWoolridge appID for FLVSS will use the range [0x0300 - 0x030F]

If you have 2 FLVSS they cannot have the samee appID.

So ..

First one : 0x0300
Second one : 0x0301

Would you please confirm that it fixes your problem?

**Comment by dWoolridge** [comment]
Thank you for the reply.  I will try and report back.

**Comment by dWoolridge** [comment]
FLVSS's have disappeared.  
I went to "Model-->Telemetry-->[mysensor]-->Device Config" and changed the first one to 0300 - 00, and the second to 0301 - 01.  Both FLVSS's stopped reporting.  I powered off the model and the X20S, and powered them back on and still no FLVSS reports.  I used "Delete All" sensors and then used "Discover new sensors" and the FLVSS are not being found anymore.  I tried this process over several power cycles with no luck.

**Comment by dWoolridge** [comment]
OK, Maybe this will help with the troubleshooting...

I disconnected all sensors EXCEPT for my FLVSS #1.  I deleted all sensors and rediscovered.  I then changed the Addr-ID to 0301-01 and confirmed after power cycling the Addr:ID assignment (by deleting/rediscovering).
![0300-01](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/16481972/e5a1fba2-6357-45f1-9c5d-f37d20fc7ca1)

I then connected ONLY FLVSS #2 and assigned it to 0302-02 and confirmed after power cycling everything.
![0302-02](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/16481972/4e839bfc-56e6-4bea-80ad-265596dda844)

When I powered off and then connected BOTH #1 and #2, the existing sensors for the 2 FLVSS no longer reported any data.  I deleted them and did rediscovery and it didn't find either.  

I then individually set them to #1: 0300-01 and #2: 0301-02.  When power cycling and plugging both sensors back in, both sensors report 0300-01 and the Sensor Conflict message is active.

So, it appears that at least one device MUST be on the base address of 0300 for it to find others.  The other sensors found then report the same addr:ID as the one on the base address.  

Summary of my complimentary troubleshooting for FrSky:
1)  To recognize multiple sensors, at least one must be on the base address, AND YOU WILL HAVE THE SENSOR CONFLICT message.  Changing either or both the address and/or ID results in the SENSOR CONFLICT message.  Both sensors will report the same addr:ID in the transmitter in this case (Model-->Telemetry-->[sensor]-->Device Config)
2)  If all sensors are moved off the base address,  it won't find any of them
